base_view_inheritance_extension/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2016 Therp BV <http://therp.nl>
```

Comment:
Both copyright for Therp BV and licence stuff can/should be changed to https. Copyright symbol should be replaced by the word Copyright.